### PR TITLE
Enable photo captions & attributions

### PR DESCRIPTION
**Pull request by kellyi**
This PR re-enables showing photo captions and attributions. The functionality to show them had been previously commented-out. I also adjusted the position of the caption/attribution labels: they now show 60px above the bottom of the screen, just above the "Swipe left to continue" label. I fixed how the labels would show and hide on taps accordingly. 

I tested it on the Nexus 7 simulator in Chrome, and the result looks like this:

screen shot 2016-03-30 at 4 11 15 pm

**Testing**
To test this in a dev env:

- get this branch
- follow the instructions in ["Developer Setup" here](https://github.com/azavea/pwd-waterworks-revealed/blob/master/README.md]) to get a local env setup.
- run `npm install phonegap -g` (probably as `sudo`)
- run `grunt app` to build the app
- run `phonegap serve` to run it on a server
- visit `localhost:3000` (or the address PhoneGap specifies) in Chrome
- turn on Chrome's Mobile Device simulator mode
- tap til you reach a map screen with a hamburger menu in the top left corner
- tap the hamburger icon, then "Settings", then turn "Enable Mock Locations" ON
- tap the hamburger, then tap "Explore the Map", then tap to move your location northwest toward the Waterworks
- tap inside one of the pink rings, then tap enter to go through the slideshow

When you reach the image screens with captions, verify that:
- the caption/attribution label appears above the "Swipe left to continue" label
- tapping the screen anywhere causes the caption label to show and hide by sliding off the bottom of the screen and by sliding up again from the bottom of the screen
- when showing the caption label again after hiding it, the label snaps back to its proper position above the "Swipe left" label
- if the caption label's shown on one screen, it will appear as shown on next screen on swiping left
- if the label's hidden on one screen, it will be hidden on the next screen -- and tapping anywhere will cause it to show again

Connects #176

### CODE REVIEW

File: plugins/browser.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (mmcfarland) [inline plugins/browser.json]: Any idea where this config file came from and if it's necessary in source control?

**Comment by kellyi** [comment]
Just checked and it looks like that file may have gotten autogenerated when I was building. I removed it and `phonegap app` still works as it should on my machine.

It might be okay to remove the other manifest files from source control, but since they're already checked in I'll just leave them there.

**Comment by mmcfarland** [comment]
The "Align Screen" instructions seem to appear in both the caption and and the regular align instruction container.  The caption with those instructions can still be toggled on tap.  Perhaps captions should be disabled on that initial screen - or the text removed and logic added to not display the caption container if it's empty?  

screenshot from 2016-03-31 09 46 31

**Comment by kellyi** [comment]
Sounds good -- I'll take a look!

**Comment by kellyi** [comment]
Looks like some extra code was required to get captions to show on the first screen beneath the instructions, so I cut those lines then added a guard statement so users can't inadvertently toggle the captions hidden or shown on the first screen and on screens with A/V elements (which lack captions).

**Comment by mmcfarland** [comment]
+1, works well.

**Pull request merged**